pkg/reconciler/workload/resource/resource_controller.go

```diff
@@ -301,19 +316,22 @@ var namespaceBlocklist = sets.NewString("kube-system", "kube-public", "kube-node
 // enqueueResourcesForNamespace adds the resources contained by the given
 // namespace to the queue if there scheduling label differs from the namespace's.
 func (c *Controller) enqueueResourcesForNamespace(ns *corev1.Namespace) error {
+	logger := logging.WithObject(logging.WithReconciler(klog.Background(), controllerName), ns).WithValues("operation", "enqueueResourcesForNamespace")
```

Comment:
don't we get a function name by default?

Reply:
Not yet, no - I think that's based on the ... formatter ... ? In a follow-up I'll get function name/filename into them all and we can remove this (and similar things).